packages/horizon/contracts/staking/HorizonStaking.sol

```diff
@@ -387,7 +387,9 @@ contract HorizonStaking is HorizonStakingBase, IHorizonStakingMain {
     ) external override notPaused {
         address verifier = msg.sender;
         Provision storage prov = _provisions[serviceProvider][verifier];
-        require(prov.tokens >= tokens, HorizonStakingInsufficientTokens(prov.tokens, tokens));
+        DelegationPoolInternal storage pool = _getDelegationPool(serviceProvider, verifier);
+        uint256 availableTokens = prov.tokens + pool.tokens;
```

Comment:
Nit, I would rename this variable just to avoid any potential confusion with actual `tokensAvailable` in a provision. Maybe `tokensSlashable` or `tokensProvisionTotal`?

Reply:
yeah, I like `tokensProvisionTotal`, will update.